lisdf/components/gui.py

```diff
@@ -36,11 +36,10 @@ def __post_init__(self) -> None:
 
     def _to_sdf(self, ctx: StringifyContext) -> str:
         name_str = f' name="{self.name}"' if self.name else ""
-        fmt = f"""<camera{name_str}>"""
-        fmt += f"""<pose>{self.pose.to_sdf(ctx)}</pose>"""
-        fmt += f"""<projection_type>{self.projection_type}</projection_type>"""
-        fmt += """</camera>"""
-        return fmt
+        return f"""<camera{name_str}>
+          <pose>{self.pose.to_sdf(ctx)}</pose>
+          <projection_type>{self.projection_type}</projection_type>
+        </camera>"""
```

Comment:
Will this result in a larger indent than you wanted?

Reply:
Thanks for catching that!! My IDE was helping me a bit too much.